app/assets/javascripts/discourse/models/user.js

```diff
@@ -859,6 +860,15 @@ const User = RestModel.extend({
 
   changeTimezone(tz) {
     this._timezone = tz;
+  },
+
+  calculateMutedIds(notificationLevel, id, type) {
+    const muted_ids = User.currentProp(type);
```

Comment:
Is there a case where this wouldn't be `this`? Not clear why you'd need to use the singleton here.